org.OpenT2T.Sample.SuperPopular.Thermostat/org.OpenT2T.Sample.SuperPopular.Thermostat.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE node PUBLIC "-//freedesktop//DTD D-BUS Object Introspection 1.0//EN" "http://standards.freedesktop.org/dbus/introspect-1.0.dtd"[]>
+<node>
+  <interface name="org.OpenT2T.Sample.SuperPopular.Thermostat">
+  
+    <!-- Turn on the HVAC -->
+    <method name="turnOn"/>
+    
+     <!-- Turn off the HVAC -->
+    <method name="turnOff"/>
+    
+    <!-- Get the current ambient temperature -->
+    <method name="getRoomTemperature">
+      <arg name="temp" type="d" direction="out" />
+    </method>
+
+     <!-- get the current cooling temperature (a.k.a. max setpoint) -->
+    <method name="getCoolingSetPoint">
```

Comment:
I'm nitpicking, but the 'P' should not be capitalized because "setpoint" as used here is one word.